qchem/pennylane_qchem/qchem.py

```diff
@@ -531,34 +531,34 @@ def generate_mol_hamiltonian(
                 map the second-quantized electronic Hamiltonian to the qubit Hamiltonian
         outpath (str): path to the directory containing output files
     Returns:
-        tuple(QubitOperator, int): the fermionic-to-qubit transformed Hamiltonian
-            and the total number of qubits
+        tuple(pennylane.beta.vqe.Hamiltonian, int): the fermionic-to-qubit transformed
+        Hamiltonian and the number of qubits
      """
 
     geometry = read_structure(mol_geo_file, outpath)
 
-    hf_data = gen_meanfield_data(
+    hf_data = meanfield_data(
         mol_name, geometry, mol_charge, multiplicity, basis_set, qc_package, outpath
     )
 
-    docc_indices, active_indices = gen_active_space(
+    docc_indices, active_indices = active_space(
         mol_name, hf_data, n_active_electrons, n_active_orbitals
     )
 
     return (
-        gen_hamiltonian_pauli_basis(mol_name, hf_data, mapping, docc_indices, active_indices),
+        decompose_hamiltonian(mol_name, hf_data, mapping, docc_indices, active_indices),
         2 * len(active_indices),
     )
 
 
 __all__ = [
     "read_structure",
-    "gen_meanfield_data",
-    "gen_active_space",
-    "gen_hamiltonian_pauli_basis",
+    "meanfield_data",
+    "active_space",
+    "decompose_hamiltonian",
     "_qubit_operator_to_terms",
     "_terms_to_qubit_operator",
     "_qubit_operators_equivalent",
```

Comment:
@ixfoduap, I vaguely remembered you saying some of these private functions might be useful for the user?

Reply:
These private functions were implemented by Zy to convert convert OpenFermion ``QubitOperator`` to PennyLane ``Hamiltonian`` operator and viceversa and for testing the function ``convert_hamiltonian``.

Reply:
:+1: